test/utils/userHelperFunctions.js

```diff
@@ -0,0 +1,10 @@
+const chai = require("chai");
+const { expect } = chai;
+
+function assertUserIds(users, ids) {
+  ids.forEach((id) => {
+    /* eslint-disable no-unused-expressions */
```

Comment:
It is necessary ?

Reply:
yeah since the line was called again and again in assertions created it into a function.